pkg/frontend/adminactions/resources_list_test.go

```diff
@@ -141,7 +150,12 @@ func TestResourcesList(t *testing.T) {
 			virtualMachines := mock_compute.NewMockVirtualMachinesClient(controller)
 			virtualNetworks := mock_network.NewMockVirtualNetworksClient(controller)
 			routeTables := mock_network.NewMockRouteTablesClient(controller)
-			tt.mocks(tt, resources, virtualMachines, virtualNetworks, routeTables)
+			diskEncryptionSets := mock_compute.NewMockDiskEncryptionSetsClient(controller)
+
+			validListByResourceGroupMock(resources)
+			validVirtualMachinesMock(virtualMachines)
+
+			tt.mocks(tt, resources, virtualMachines, virtualNetworks, routeTables, diskEncryptionSets)
```

Comment:
Cleanup: It looks like we no longer need to pass `tt, resources, virtualMachines` becuase we moved some things out of `mocks` function.